tests/cli/build/build_tests.py

```diff
@@ -70,12 +69,12 @@ def test_001_build_android(self):
         assert not File.exists(os.path.join(TnsPaths.get_platforms_android_folder(self.app_name), '*.ios.js'))
 
         # Verify apk does not contain aar files
-        archive = os.path.join(TnsPaths.get_apk_path(self.app_name), self.debug_apk)
-        File.unzip(archive, 'temp')
+        apk_path = TnsPaths.get_apk_path(app_name=self.app_name, release=False)
+        File.unzip(apk_path, 'temp')
         # Clean META-INF folder. It contains com.android.support.... files which are expected to be there due to
         # https://github.com/NativeScript/nativescript-cli/pull/3923
-        Folder.clean(os.path.join(self.app_name, 'temp', 'META-INF'))
-        temp_folder = os.path.join(self.app_name, 'temp')
+        Folder.clean(os.path.join(self.app_path, 'temp', 'META-INF'))
+        temp_folder = os.path.join(self.app_path, 'temp')
```

Comment:
You can declare temp_folder earlier and use it everywhere instead of 'temp' in some methods and temp_folder in other.

Reply:
It was so even before my commit.
It only fix the problem when you run tests and cwd is not project root.
Thanks for the comment, I will refactor it.